DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/bridging_packet_types.py

```diff
@@ -0,0 +1,264 @@
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcMac": srcMac,
+        "dstMac": self_mac,
+        "frameSize": 96,
+        "protocol": "0x0806",
+        "type" :"raw"
+    },
+    "IPv4_Broadcast": {
+        "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcIp": "1.1.1.2",
+        "dstIp": "100.1.1.255",
+        "srcMac": srcMac,
+        "dstMac": "FF:FF:FF:FF:FF:FF",
+        "frameSize": 96,
+        "protocol": "0x0800",
+        "type" :"raw"
+    },
+    "IPV4_SSH": {
+        "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcIp": "1.1.1.2",
+        "dstIp": "100.1.1.253",
+        "srcMac": srcMac,
+        "dstMac": self_mac,
+        "ipproto": "tcp",
+        "srcPort": str(randrange(0xffff + 1)),
+        "dstPort": "22",
+        "frameSize": 96,
+        "protocol": "0x0800",
+        "type" :"raw"
+    },
+    "IPV4_Telnet": {
+        "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcIp": "1.1.1.2",
+        "dstIp": "100.1.1.253",
+        "srcMac": srcMac,
+        "dstMac": self_mac,
+        "ipproto": "tcp",
+        "srcPort": str(randrange(0xffff + 1)),
+        "dstPort": "23",
+        "frameSize": 96,
+        "protocol": "0x0800",
+        "type" :"raw"
+    },
+    "Default_IPv4": {
+        "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcIp": "192.168.1.1",
+        "dstIp": "192.168.1.253",
+        "srcMac": srcMac,
+        "dstMac": self_mac,
+        "ipproto": "tcp",
+        "srcPort": str(randrange(0xffff + 1)),
+        "dstPort": "23",
+        "frameSize": 96,
+        "protocol": "0x0800",
+        "type" :"raw"
+    },
+    "IPv4_ICMP_Request": {
+        "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcIp": "1.1.1.2",
+        "dstIp": "100.1.1.253",
+        "srcMac": srcMac,
+        "dstMac": self_mac,
+        "ipproto": "icmpv1",
+        "frameSize": 96,
+        "protocol": "0x0800",
+        "type" :"raw"
+    },
+    "IPv4_DCHP_BC": {
+        "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcIp": "0.0.0.0",
+        "dstIp": "255.255.255.255",
+        "srcMac": srcMac,
+        "dstMac": "FF:FF:FF:FF:FF:FF",
+        "frameSize": 346,
+        "ipproto": "udp",
+        "srcPort": "67",
+        "dstPort": "68",
+        "frameSize": 346,
+        "protocol": "0x0800",
+        "type" :"raw"
+    },
+    "IPv4_Reserved_MC": {
+        "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        "srcIp": "1.1.1.2",
+        "dstIp": "224.0.0.69",
```

Comment:
dstIp and dstMac are connected in multicast. You cannot increment MAC separately from IP.